Sources/NIOHTTP2/HTTP2ChannelHandler+InlineStreamMultiplexer.swift

```diff
@@ -238,8 +238,36 @@ extension NIOHTTP2Handler {
             self.inbound = inboundStreamChannels
         }
 
+        /// Create a stream channel initialized with the provided closure
         public func createStreamChannel<OutboundStreamOutput>(_ initializer: @escaping NIOHTTP2Handler.StreamInitializerWithOutput<OutboundStreamOutput>) async throws -> OutboundStreamOutput {
             return try await self.inlineStreamMultiplexer.createStreamChannel(initializer).get()
         }
+
+
+        /// Create a stream channel initialized with the provided closure and return it wrapped within a `NIOAsyncChannel`.
+        /// - parameters:
+        ///     - streamInboundType: The ``NIOAsyncChannel/inboundStream`` message type for the created channel.
+        ///     This type must match the `InboundOut` type of the final handler added to the stream channel by the `initializer`
+        ///     or ``HTTP2Frame/FramePayload`` if there are none.
+        ///     - streamOutboundType: The ``NIOAsyncChannel/outboundWriter`` message type for the created channel.
+        ///     This type must match the `OutboundIn` type of the final handler added to the stream channel by the `initializer`
+        ///     or ``HTTP2Frame/FramePayload`` if there are none.
+        ///     - initializer: A callback that will be invoked to allow you to configure the
+        ///         `ChannelPipeline` for the newly created channel.
```

Comment:
typo and formatting fixes:

```suggestion
        /// Create a stream channel initialized with the provided closure and return it wrapped within a `NIOAsyncChannel`.
        ///
        /// - Parameters:
        ///     - inboundType: The ``NIOAsyncChannel/inboundStream`` message type for the created channel.
        ///       This type must match the `InboundOut` type of the final handler added to the stream channel by the `initializer`
        ///       or ``HTTP2Frame/FramePayload`` if there are none.
        ///     - outboundType: The ``NIOAsyncChannel/outboundWriter`` message type for the created channel.
        ///       This type must match the `OutboundIn` type of the final handler added to the stream channel by the `initializer`
        ///       or ``HTTP2Frame/FramePayload`` if there are none.
        ///     - initializer: A callback that will be invoked to allow you to configure the
        ///         `ChannelPipeline` for the newly created channel.
```